dev-packages/browser-integration-tests/suites/tracing/browserTracingIntegration/interactions/test.ts

```diff
@@ -70,12 +70,12 @@ sentryTest(
 
     const url = await getLocalTestPath({ testDir: __dirname });
     await page.goto(url);
-    await getFirstSentryEnvelopeRequest<Event>(page);
+    await getFirstSentryEnvelopeRequest<SentryEvent>(page);
 
     for (let i = 0; i < 4; i++) {
       await wait(100);
       await page.locator('[data-test-id=interaction-button]').click();
-      const envelope = await getMultipleSentryEnvelopeRequests<Event>(page, 1);
+      const envelope = await getMultipleSentryEnvelopeRequests<SentryEvent>(page, 1);
```

Comment:
To avoid flakes I'd recommend 
1. start listening for the envelope by calling getMultipleSentryEnvelopeRequests and storing the promise
2. clicking
3. awaiting the promise.